src/mmw/js/src/modeling/controllers.js

```diff
@@ -35,26 +38,29 @@ var ModelingController = {
                 name: 'Untitled Project',
                 created_at: Date.now(),
                 area_of_interest: App.map.get('areaOfInterest'),
-                scenarios: new models.ScenariosCollection([
-                    new models.ScenarioModel({
-                        name: 'Current Conditions',
-                        is_current_conditions: true,
-                        active: true
-                    })
-                ])
+                scenarios: new models.ScenariosCollection()
+            });
```

Comment:
For readability, consider condensing this function to something like...

    var project = this.getProject(projectId);
    App.currProject = project;
    // Wire up events...
    initViews(project);